orttraining/orttraining/python/training/_ortmodule_graph_execution_manager.py

```diff
@@ -82,6 +82,10 @@ def __init__(self, module):
         self._save_onnx = False
         self._save_onnx_prefix = ''
 
+        # Graph transformer config
+        self._propagate_cast_ops_level = -1
```

Comment:
Can we lift the scope of this config beyond Cast propagation? This level can be used to define the scope of certain graph transformations, eg the Simplified layer norm transform for HF T5 fp16 that modifies the compute precision from the model code specification. @satyajandhyala @SherlockNoMad @mrry

Reply:
Because Cast Propagation is experimental we want to have a separate control over it using a config variable. If this is something useful/required overall graph transformations then it may be implemented seperately.